database/insertData/03_template_D_basicReviewTesting.js

```diff
@@ -266,29 +266,61 @@ exports.queries = [
           }
           templatePermissionsUsingId: {
             create: [
-              { id: 4000, permissionNameId: 2000 }
               {
-                id: 4001
-                permissionNameId: 5000
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "applyGeneral" }
+                }
+              }
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewReviewTestScreeningSection1" }
+                }
                 stageNumber: 1
                 levelNumber: 1
                 restrictions: { templateSectionRestrictions: ["S1"] }
               }
               {
-                id: 4002
-                permissionNameId: 5001
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewReviewTestScreeningSection2" }
+                }
                 stageNumber: 1
                 levelNumber: 1
                 restrictions: { templateSectionRestrictions: ["S2"] }
               }
-              { permissionNameId: 10500, restrictions: { canSelfAssign: true }, stageNumber:1, level: 1 }
-              { id: 4003, permissionNameId: 6000, stageNumber: 2, levelNumber: 1 }
-              { id: 4004, permissionNameId: 6001, stageNumber: 2, levelNumber: 2 }
-              { id: 4005, permissionNameId: 7000, stageNumber: 3, levelNumber: 1 }
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewSelfAssignable" }
+                }
+                restrictions: { canSelfAssign: true }
+                stageNumber: 1
+                levelNumber: 1
+              }
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewReviewTestAssessmentLvl1" }
+                }
+                stageNumber: 2
+                levelNumber: 1
+              }
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewReviewTestAssessmentLvl2" }
+                }
+                stageNumber: 2
+                levelNumber: 2
+              }
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewReviewTestApproval" }
+                }
+                stageNumber: 3
+                levelNumber: 1
+              }
```

Comment:
The permission name called "reviewSelfAssignable" has that restriction, should all the Stage 2 ones be self-assignable too?

Reply:
I think that's the current only way to select a Review to be started for consolidation, since we still have a pending issue for selecting a consolidator (lvl 2) once the consolidation can be started.

Reply:
Okay, let's just add them in as we work on Consolidation, as I'm a little unclear how this is enforced, and we'll be changing the location of these in #299 anyway.